core/src/main/java/org/elasticsearch/indices/store/IndicesStore.java

```diff
@@ -97,11 +99,12 @@ public void clusterChanged(ClusterChangedEvent event) {
         }
 
         for (IndexRoutingTable indexRoutingTable : event.state().routingTable()) {
+            IndexSettings indexSettings = new IndexSettings(event.state().getMetaData().index(indexRoutingTable.index()), settings, Collections.EMPTY_LIST);
```

Comment:
ack :)
> On 02 Dec 2015, at 16:43, Jason Tedor <notifications@github.com> wrote:
> 
> In core/src/main/java/org/elasticsearch/indices/store/IndicesStore.java:
> 
> > @@ -97,11 +99,12 @@ public void clusterChanged(ClusterChangedEvent event) {
> >          }
> >  
> >          for (IndexRoutingTable indexRoutingTable : event.state().routingTable()) {
> > +            IndexSettings indexSettings = new IndexSettings(event.state().getMetaData().index(indexRoutingTable.index()), settings, Collections.EMPTY_LIST);
> 
> nit: Collections.emptyList() to avoid unchecked assignment warnings.
> 
> See #15187.
> 
> —
> Reply to this email directly or view it on GitHub.
>